bin/zookeeper-server-start.sh

```diff
@@ -1,44 +0,0 @@
-#!/bin/bash
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-if [ $# -lt 1 ];
-then
-	echo "USAGE: $0 [-daemon] zookeeper.properties"
-	exit 1
-fi
-base_dir=$(dirname $0)
-
-if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
-    export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/log4j.properties"
-fi
-
-if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
-    export KAFKA_HEAP_OPTS="-Xmx512M -Xms512M"
-fi
-
-EXTRA_ARGS=${EXTRA_ARGS-'-name zookeeper -loggc'}
-
-COMMAND=$1
-case $COMMAND in
-  -daemon)
-     EXTRA_ARGS="-daemon "$EXTRA_ARGS
-     shift
-     ;;
- *)
-     ;;
-esac
-
-exec $base_dir/kafka-run-class.sh $EXTRA_ARGS org.apache.zookeeper.server.quorum.QuorumPeerMain "$@"
```

Comment:
After removing this script, we won't be able to use `DEV_BRANCH` to run the ZooKeeper service in E2E tests. Perhaps, for now, we could fix the ZooKeeper version to `3.8` instead of using `DEV_BRANCH`?

Reply:
Let's do the ducktape stuff in a separate commit. We still have to figure out stuff like how we're going to run Kafka 2.1, so maybe we still need zk.sh for that. I've proposed downgrading the ZK used by Vagrant / ducker-ak to a version which supports all the ancient Kafkas (2.1, 2.2, 2.3, 2.4, 2.5, 2.6, etc.)

Reply:
> Let's do the ducktape stuff in a separate commit. We still have to figure out stuff like how we're going to run Kafka 2.1, so maybe we still need zk.sh for that. I've proposed downgrading the ZK used by Vagrant / ducker-ak to a version which supports all the ancient Kafkas (2.1, 2.2, 2.3, 2.4, 2.5, 2.6, etc.)

yes, you are right that we can address that in a separate commit. I'm +1 to this PR to remove all zk-related code from ducktape if we can use kafak 3.9 release to run zk 3.8 cluster.

Reply:
open https://issues.apache.org/jira/browse/KAFKA-18004 for using 3.x release run zookeeper service